examples/knative/go/kn-go-tagging/go.mod

```diff
@@ -0,0 +1,12 @@
+module github.com/vmware-samples/vcenter-event-broker-appliance/examples/knative/go/kn-go-tagging
+
+go 1.16
+
+require (
+	github.com/cloudevents/sdk-go/v2 v2.4.1
+	github.com/kelseyhightower/envconfig v1.4.0
+	github.com/vmware/govmomi v0.25.0
```

Comment:
nit: you might want to use a more recent version, shouldn't break anything

Reply:
I'm on overtime at the moment, so I'll upgrade, but I'm not going to build another image to test. I'm going to take the risk and hope updating govmomi to v0.27.2 didn't break anything.